numpy/core/src/umath/ufunc_object.c

```diff
@@ -479,8 +480,26 @@ _is_alnum_underscore(char ch)
     return _is_alpha_underscore(ch) || (ch >= '0' && ch <= '9');
 }
 
+NPY_NO_EXPORT npy_longlong
+npy_strtoll(const char *str, char **endptr, int base);
 /*
- * Return the ending position of a variable name
+ * Convert a string into a number
+ */
+static npy_longlong
+_get_size(const char* str)
```

Comment:
This function should return and intp, and verify that the result fits in an intp, given that it's always cast when called.

Reply:
done

Reply:
fixed

Reply:
Seems like the signature is still `npy_int` rather than `npy_intp`?